source/neuropod/bindings/java/Makefile

```diff
@@ -0,0 +1,9 @@
+export MY_JNI_HEADERS=${PWD}/src/main/native
```

Comment:
We shouldn't have makefiles anywhere.

Can we just generate the headers and check them in?

Since the implementation isn't autogenerated, I think generating the headers during the build might cause more problems than it's worth